docs/user_guide/5_changelog.md

```diff
@@ -2,6 +2,31 @@
 
 
 ## ROS Noetic
+
+### Version 1.0.18 (current noetic-release)
+* Fixing demo behaviour when tactile sensors are installed (1)
+* Fixing demo behaviour when tactile sensors are installed (2)
+
+### Version 1.0.17
+* Update repository.rosinstall
```

Comment:
In what repo, what's the update (My guess, it's the dependency on sr_ethercat_config (sr_config?))

Reply:
Improved